quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/LoggingPropertyMappers.java

```diff
@@ -2,23 +2,37 @@
 
 import static org.keycloak.quarkus.runtime.integration.QuarkusPlatform.addInitializationException;
 
+import java.io.File;
+import java.util.List;
 import java.util.Locale;
 import java.util.function.BiFunction;
 import java.util.logging.Level;
 
 import org.jboss.logmanager.LogContext;
+import org.keycloak.quarkus.runtime.Environment;
 import org.keycloak.quarkus.runtime.Messages;
 
 import io.smallrye.config.ConfigSourceInterceptorContext;
 
-final class LoggingPropertyMappers {
+public final class LoggingPropertyMappers {
 
     private static final String DEFAULT_LOG_LEVEL = "info";
+    private static final String DEFAULT_LOG_HANDLER = "console";
+    private static final String DEFAULT_LOG_FILENAME = "keycloak.log";
+    public static final String DEFAULT_LOG_PATH = "data" + File.separator + "log" + File.separator + DEFAULT_LOG_FILENAME;
+    private static final List<String> AVAILABLE_LOG_HANDLERS = List.of(DEFAULT_LOG_HANDLER,"file");
+    private static final String DEFAULT_CONSOLE_OUTPUT = "default";
 
     private LoggingPropertyMappers(){}
 
     public static PropertyMapper[] getMappers() {
         return new PropertyMapper[] {
+                builder().from("log")
+                        .defaultValue(DEFAULT_LOG_HANDLER)
+                        .description("Enable one or more log handlers in a comma-separated list. Available log handlers are: " + String.join(",", AVAILABLE_LOG_HANDLERS))
+                        .paramLabel("<handler>")
+                        .expectedValues("console","file","console,file","file,console")
```

Comment:
We should look at how to avoid this but I understand it can not block this PR. Going to create an issue for this.